2-mandatory/3-stocks.js

```diff
@@ -33,8 +33,25 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Solve the smaller problems, and then build those solutions back up to solve the larger problem.
         Functions can help with this!
 */
+
+// 
+
```

Comment:
This is a great solution! 👍 
I love the use of the extra function here 😄 It makes the code very easy to read and understand.